adapters/upload_tool.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * This code is responsible for implementing all methods related to fetching
+ * and returning data for the data uploaded via the OpenAQ Upload Data tool
+ *
+ */
+'use strict';
+
+import { S3 } from 'aws-sdk';
+import csv from 'csv-parser'
+import log from '../lib/logger';
+
+// The S3 bucket containing the data is in a different region
+const s3 = new S3({region: 'us-west-2'});
+
+const UPLOAD_TOOL_BUCKET = 'upload-tool-bucket-dev'
```

Comment:
Note: this currently expects the final S3 bucket for the files to exist in upload-tool-bucket-dev. This bucket needs to be changed to match what is in the [Upload tool](https://github.com/openaq/openaq-upload/pull/24) repo for production stage.